tests/integrated/test-squash/runtest

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import netCDF4 as nc
+import itertools as it
+import os
+
+
+def run(cmd):
+    s = os.system(cmd)
+    if s:
+        raise RuntimeError("%s exited with non-zero %d value" % (cmd, s))
+
+
+def verify(f1, f2):
+    org = nc.Dataset(f1)
+    para = nc.Dataset(f2)
```

Comment:
Please could you rename these? I assume "org" means "original", but I have no clue what "para" means

Reply:
was parallel, but has no meaning any more. I switched to `d1` and `d2`.